app/src/main/java/ca/lakeeffect/scoutingapp/Field.java

```diff
@@ -58,7 +69,7 @@ public Field(SurfaceView s, Bitmap fieldRed, Bitmap fieldBlue) {
         this.fieldRed = fieldRed;
         this.fieldBlue = fieldBlue;
 
-        normal.setColor(Color.RED);
+        normal.setColor(s.getResources().getColor(R.color.colorPrimary));
```

Comment:
I think this function is actually deprecated. See https://stackoverflow.com/a/5271452/1985387